src/System.Text.Json/src/System/Text/Json/Serialization/JsonNamingPolicy.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Determines the naming policy used to convert a JSON name to another format, such as a camel-casing format.
+    /// </summary>
+    public abstract class JsonNamingPolicy
+    {
+        protected JsonNamingPolicy() { }
+
+        /// <summary>
+        /// Returns the naming policy for camel-casing.
+        /// </summary>
+        public static JsonNamingPolicy CamelCase { get; } = new JsonCamelCaseNamePolicy();
+
+        /// <summary>
+        /// Converts the provided name.
```

Comment:
@mairaw, @rpetrusha: what type of documentation should be added in source for cases like this. This seems too terse or not useful to add here.

Reply:
For abstract members of abstract classes, we start the description with "When overridden in a derived class,..."

You could say what the method converts the provided (or specified is preferred) name to?